publish-imgs

```diff
@@ -19,7 +14,9 @@ calcout=$(echo $derivation | jq '.[] | .env.out')
 calcout=$(basename ${calcout} | cut -d '-' -f1)
 registry_endpoint="${registry}/${org}/${image_name}:${calcout}"
 
-# Check if this image is already in the registry
+# Check if this image is already in the registry and exit if so
 skopeo inspect docker://${registry_endpoint} && exit 0
 
+nix-build -A imgs.$1
```

Comment:
Lets build the image only when its not in the registry

Reply:
thanks for adding this in here so we dont have to build unnecessarily 👍

Reply:
Could @sarcasticadmin clarify on the syntax here `imgs.$1`? I know we are passing in the image name through the CI, but why the `imgs.` before it? Is that a nix specific syntax nuance?

Reply:
The syntax is due to the `import` at the top-level here: https://github.com/Nebulaworks/nix-garage/pull/19/files#diff-5712e736e0de6ba170577f8472c398e9R7 all things defined in `imgs/default.nix` will now have an attribute prefix of `imgs.<pkg>`. This is separate from the `overlay.nix` thats defined earlier so sorry if theres confusion there. I didnt feel that it would be useful for use to have the docker images included in the overlay.

Reply:
Ah I see the relationship now that makes a lot more sense, that's a nice way to easily tie in the entire image directory into the `default.nix` derivation.